modules/perception/obstacle/camera/visualizer/glfw_fusion_viewer.cc

```diff
@@ -63,6 +63,13 @@ std::vector<std::vector<int>> GLFWFusionViewer::s_color_table = {
     std::vector<int>{255, 0, 255}, std::vector<int>{255, 255, 0},
     std::vector<int>{255, 128, 0}};
 
+std::vector<cv::Scalar> lane_map_colors = {
+    cv::Scalar(0, 0, 0), cv::Scalar(100, 240, 0), cv::Scalar(150, 180, 0),
```

Comment:
Please make them to have one color in one line and add color name (at least something similar)

Reply:
Semantic annotation (L0, R0, ... ) will be also useful